docs/source/examples/BSTLD.rst

```diff
@@ -174,106 +166,83 @@ structure for ``BSTLD`` should be like:
 
 .. _graviti opendatset platform: https://www.graviti.cn/open-datasets
 
-TensorBay supplies two methods to fetch :ref:`basic_concepts:Segment` from
-:ref:`basic_concepts:Dataset`.
-
-.. code:: python
+TensorBay supplies two methods to fetch :ref:`reference/dataset_structure:Segment` from
+:ref:`reference/dataset_structure:Dataset`.
 
-    >>> train_segment = dataset.get_segment_by_name("train")
-
-    >>> first_segment = dataset[0]
-
-The :ref:`basic_concepts:Segment` you get now is the same as the one you read from TensorBay in the
-:ref:`quick_start:Read Dataset from TensorBay` part.
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 20-21
 
-**************************
- Write Dataset Dataloader
-**************************
+The :ref:`reference/dataset_structure:Segment` you get now is the same as the one you read from TensorBay in the
+:ref:`quick_start/getting_start_with_tensorbay:Read Dataset from TensorBay` part.
 
-If there is no :ref:`contribution:Dataloader` available to your target dataset,
-you can write one and contribute it.
-Now we take ``BSTLD`` as an example to explain how to write a dataloader for datasets with Box2D
-labels.
+*****************************
+ Upload images to the Dataset
+*****************************
 
-Write the Catalog
-=================
+Once you write your own :ref:`reference/glossary:Dataloader` and read the local dataset into a
+:ref:`reference/dataset_structure:Dataset` object, you can upload it to TensorBay to share with the community.
 
-Before writing the dataloader, we first need to write the :ref:`contribution:Catalog`.
-Catalog is a json file contains all label information of one dataset.
-See :ref:`this page <basic_concepts:Catalog & SubCatalog>` for more details.
-The only annotation type for ``BSTLD`` is :ref:`supported_label_types:Box2D`, and there are 13
-:ref:`supported_label_types:Category` types and one :ref:`supported_label_types:Attributes` type.
+.. literalinclude:: ../../../examples/bstld.py
+   :language: python
+   :lines: 23-24
 
-.. literalinclude:: ../../../tensorbay/opendataset/BSTLD/catalog.json
-   :language: json
-   :name: BSTLD-catalog
-   :linenos:
+*****************************
+ Read images from the Dataset
+*****************************
 
+As mentioned in :ref:`quick_start/getting_start_with_tensorbay:Getting start with TensorBay`,
+obtain_ and fork_ should be done before reading a dataset from TensorBay. Then,
+pass the correct dataset name to the GAS client, and you will get a
+:ref:`features/tensorbay_client:Dataset Client`.
 
-Write the Dataloader
-====================
+.. _fork: https://docs.graviti.cn/guide/opendataset/fork
 
-The function of :ref:`contribution:Dataloader` is to read the dataset into a
-:ref:`basic_concepts:Dataset` object.
-The :ref:`code block <BSTLD-dataloader>` below displays the ``BSTLD`` dataloader.
+.. _obtain: https://docs.graviti.cn/guide/opendataset/get
 
```

Comment:
First of all, create a dataset client by passing the dataset name to the GAS client.